### PR TITLE
Change in case we enable the sec vtx in online

### PR DESCRIPTION
**Pull request by chiarazampolli**
In online, the strangeness tracking would be off via export ARGS_EXTRA_PROCESS_o2_secondary_vertexing_workflow='--disable-cascade-finder --disable-3body-finder --disable-strangeness-tracker but then the ASYNC part of the FST would fail.

See https://github.com/AliceO2Group/O2DPG/pull/1834

### CODE REVIEW

File: prodtests/full_system_test.sh [hunk 1]
```diff
@@ -255,6 +258,8 @@ for STAGE in $STAGES; do
     export HOSTMEMSIZE=$TPCTRACKERSCRATCHMEMORY
     export CTFINPUT=0
     export WORKFLOW_PARAMETERS="${WORKFLOW_PARAMETERS},CALIB,CTF,EVENT_DISPLAY,${FST_SYNC_EXTRA_WORKFLOW_PARAMETERS}"
+    # enabling SECVTX
```
Comment (chiarazampolli) [inline prodtests/full_system_test.sh:263]: Hello @shahor02 , 

In https://github.com/AliceO2Group/O2DPG/pull/1834 you proposed to enable the SECVTX in the FST. Is it ok here? Or you would add it also somewhere in the "WITHGPU" part?

Chiara

Comment (shahor02) [reply prodtests/full_system_test.sh:263]: The QC is running also in the `WITHGPU`, so I would add it also there.

Comment (chiarazampolli) [reply prodtests/full_system_test.sh:263]: Done

**Comment by chiarazampolli** [comment]
Hello @shahor02 , 

Is there a merge conflict? This is what is reported by fullCI_slc9, but I don't see any log.

Chiara

**Comment by shahor02** [comment]
I guess there is a problem with the CI builder, pinging to @ktf

**Comment by ktf** [comment]
@singiamtel could you have a look when you are back? It looks like the new tester machine is confused.

**Pull request merged**